server/domain/academic-record/models/AcademicSummary.ts

```
@@ -0,0 +1,43 @@
+  }
+
+  getRemainingCredits(totalRequiredCredits: number): number | null {
+    return this.totalEarnedCredits ? Math.max(0, totalRequiredCredits - this.totalEarnedCredits) : null;
+  }
+
+  getCompletionRate(totalRequiredCredits: number): number | null {
+    return this.totalEarnedCredits ? (this.totalEarnedCredits / totalRequiredCredits) * 100 : null;
+  }
+
+  isHonorsEligible(): boolean {
+    if (!this.cumulativeGpa) return false;
```

ESLint

[error] Expected { after 'if' condition.

(curly)